draft-ietf-taps-impl.md

```diff
@@ -770,8 +770,8 @@ For Protocol-specific Properties, such as the User Timeout that applies to TCP,
 If an error is encountered in setting a property (for example, if the application tries to set a TCP-specific property on a Connection that is
 not using TCP), the action should fail gracefully. The application may be informed of the error, but the Connection itself should not be terminated.
 
-The Transport Services API should allow protocol instances in the Protocol Stack to pass up arbitrary generic or protocol-specific
-errors that can be delivered to the application as Soft Errors. These allow the application to be informed of ICMP errors, and other similar events.
+The Transport Services implementation SHOULD allow protocol instances in the Protocol Stack to pass up arbitrary generic or protocol-specific
```

Comment:
could say:/A Transport Services implementation  is RECOMMENDED to allow protocol instances in the Protocol Stack to pass up arbitrary generic or protocol-specific/